package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
Why is there a package-lock file in the root? Can we get rid of it?

Reply:
hmmm... considering there's a package-lock with actual contents in the frontend folder, i think it's safe to remove this

Reply:
Removed in 0088853